day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,12 @@
+monkey = ["Ten", "Nine", "Eight", "Seven", "Six", "Five","Four", "Three", "Two", "One"]
+monkey.each do |ele|
+  1.times {p "#{ele} little monkey(s) jumping on the bed, One fell off and bumped his head, Mama called the doctor and the doctor said,  'No more monkeys jumping on the bed!'"}
```

Comment:
what happens if you take away the times block here?

Reply:
It will still work fine and now I believe there was no need of the times block.